docs/alert_config.md

```diff
@@ -0,0 +1,62 @@
+# Observability Pack Alert Configuration
+
+**Title:** Observability Pack Alert Configuration
+
+| Type                      | `object`                                                                  |
+| ------------------------- | ------------------------------------------------------------------------- |
+| **Additional properties** | [[Any type: allowed]](# "Additional Properties of any type are allowed.") |
+|                           |                                                                           |
```

Comment:
Looks like our script is adding an extra row at the bottom of these tables.

Reply:
i double checked through their documentation & don't see an option to disable that :rip: